tests/testthat/test-assortativity.R

```diff
@@ -13,29 +13,27 @@ test_that("assortativity works", {
     num / sqrt(den1) / sqrt(den2)
   }
 
-  asd <- assortativity_degree(g)
-  as <- assortativity(g, values = degree(g, mode = "out"), values.in = degree(g, mode = "in"))
-  as2 <- assR(g)
+  assortativity_igraph <- assortativity(
+    g,
+    values = degree(g, mode = "out"),
+    values.in = degree(g, mode = "in")
+  )
 
-  expect_that(asd, equals(as))
-  expect_that(asd, equals(as2))
+  expect_equal(assortativity_degree(g), assortativity_igraph)
```

Comment:
is it bad to run `assortativity_degree(g)` twice? I find that more readable than giving it a name, that the reader has to keep track of.

Reply:
Running twice is fine.